pytools/persistent_dict.py

```diff
@@ -51,8 +51,10 @@
     cast,
 )
 
-from siphash24 import siphash13
-
+try:
+    from siphash24 import siphash13 as _DEFAULT_HASH
+except ImportError
+    from hashlib import sha256 as _DEFAULT_HASH
```

Comment:
Could you add a warning in this case that the recommended hash was not found?

Reply:
Done